skyportal/tests/api/test_thumbnail.py

```diff
@@ -25,6 +26,16 @@ def test_token_user_post_get_thumbnail(upload_data_token, public_group, ztf_came
     assert status == 200
     assert data['data']['id'] == obj_id
 
+    # wait for the thumbnails to populate
+    nretries = 0
+    thumbnails_loaded = False
+    while not thumbnails_loaded and nretries < 5:
+        thumbnails = DBSession.query(Obj).filter(Obj.id == obj_id).first().thumbnails
```

Comment:
May as well do this with `select` and a context manager?

Reply:
yup